bin/rdme

```diff
@@ -2,13 +2,25 @@
 const chalk = require('chalk');
 const core = require('@actions/core');
 
+const nodeVersion = require('parse-node-version')(process.version);
 const updateNotifier = require('update-notifier');
 const pkg = require('../package.json');
 
 const isGHA = require('../src/lib/isGitHub');
 
 updateNotifier({ pkg }).notify();
 
+/**
+ * We use optional chaining throughout the library, which doesn't work on Node 12, so to curb
+ * support questions about why rdme is throwing an "Unexpected token '.'" error we should hard
+ * stop if we're on any Node version that doesn't support it.
+ */
+if (nodeVersion.major <= 12) {
```

Comment:
Since we already have `semver` in the mix, could we use the `engines.node` field and have `semver` check if the current version satisfies the requirements?